src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -1073,8 +1073,14 @@ public class ADDR_Addresses_TDTM extends TDTM_Runnable {
         }
         
         // if specified, also include Address Type
-        if (strAddrType != null) 
+        if (strAddrType != null) {
             addr.Address_Type__c = strAddrType;
+        } else {
+            // A null value will force a newly created Address record to use its defined default value for
+            // the picklist, which if not reflected here could cause isAddressChanged() to return true
+            // in cases when it should not.
+            addr.Address_Type__c = defaultAddressType;
```

Comment:
@force2b , is this fix related or already fixed by https://github.com/SalesforceFoundation/Cumulus/pull/2635 ?